automation/run-integration-tests.sh

```diff
@@ -103,6 +103,14 @@ mkdir -p $EXPORT_DIR
 CONTAINER_ID="$(docker run --privileged -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro -v $PROJECT_PATH:/workspace/nmstate -v $EXPORT_DIR:$CONT_EXPORT_DIR $DOCKER_IMAGE)"
 [ -n "$debug_exit_shell" ] && trap open_shell EXIT || trap run_exit EXIT
 docker_exec 'while ! systemctl is-active dbus; do sleep 1; done'
+docker_exec '
+    systemctl start NetworkManager.service
```

Comment:
Can't you use a simple retry like in line 105? Let travis-ci time-out if it does not succeed.